spec/requests/api/v1.0/portfolios_spec.rb

```diff
@@ -422,6 +422,18 @@
       end
     end
 
+    shared_examples_for "bad_tags" do
+      let(:bad_tags) { %w[/ /approval /approval/workflows=] }
```

Comment:
I like this shared example a lot.  I also realize we do test 'good_tags' in the shared example for `tag_add_test`.
Do you see an value in creating a `good_tags` shared example that builds out some obscure good tags? Potential overkill but it seemed like a good way to validate the regex pattern from both sides of the coin.

Reply:
I like that idea a lot - adding it now.